### PR TITLE
Forcefield update

### PR DESCRIPTION
**Pull request by florianjoerg**
## Description
Updated force field files
Made some clean up

## Todos
  - [x] Get new force field files
  - [x] Clean up old force field directories

## Status
- [x] Ready to go

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -106,5 +106,5 @@ ENV/
 *profraw
 
 # user defined (flo)
-protex/charmm-ff/out/
-protex/charmm-ff/traj/
+#protex/charmm-ff/out/
+#protex/charmm-ff/traj/
```
Comment (wiederm) [inline .gitignore]: this seems like two directories that we want to ignore?

File: charge_changes.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{"dcd_save_freq": 100,
```
Comment (wiederm) [inline charge_changes.json]: do you want to commit this file?

File: drude_temp.out [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#"Step"	"T_COM"	"T_Atom"	"T_Drude"	"KE_COM"	"KE_Atom"	"KE_Drude"
```
Comment (wiederm) [inline drude_temp.out]: do you want to commit this file?

**Review by wiederm — APPROVED** [review]
I would remove all *dcd, *out, *dat files and also add these endings to the `.gitignore` file. I don't think that we need them, right?
Otherwise looks good to me, this can be merged!

File: .gitignore (hunk 1 above)
Comment (florianjoerg) [reply .gitignore]: Yes, but these directories do not exist anymore, so we can delete it.

File: charge_changes.json (hunk 2 above)
Comment (florianjoerg) [reply charge_changes.json]: No, I do not think it is necessary, at least not now

File: drude_temp.out (hunk 3 above)
Comment (florianjoerg) [reply drude_temp.out]: No we can remove it as well

**Pull request merged**